Source/WebKit/WPEPlatform/wpe/drm/WPEScreenDRM.cpp

```diff
@@ -113,6 +113,33 @@ const WPE::DRM::Crtc wpeScreenDRMGetCrtc(WPEScreenDRM* screen)
     return *screen->priv->crtc;
 }
 
+double wpeScreenDRMGuessScale(WPEScreenDRM* screen)
+{
+    static constexpr uint32_t minimum2xScaleDPI = 2 * 96;
+    static constexpr uint32_t minimum2xPixelHeight = 1200;
+    static constexpr float mmPerInch = 25.4;
```

Comment:
I would add a comment here explaining the logic we are following to decide the scale.

Reply:
Will do 👍🏼

Reply:
Comment added.